haas/ext/obm/ipmi_1_5.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2015-2016 Massachusetts Open Cloud Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the
+# License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS
+# IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""IPMI driver for implementing out of band management. """
+
+from sqlalchemy import Column, String, Integer, ForeignKey
+import schema
+import subprocess
+
```

Comment:
"import logging" is missing here 
and also in ipmi file. 
Can you also add this in the ipmi.py also ?